portfolio/src/main/webapp/index.html

```diff
@@ -37,16 +37,21 @@ <h2>Comments</h2>
     </form>
     <br>
     <form>
-      <label>Display the latest comments </label>
+      <label>Display the most popular comments </label>
       <select name="limit" id="limit">
         <option value="3" selected="selected">3</option>
         <option value="10">10</option>
         <option value="20">20</option>
       </select>
       <button type="button" onclick="getComments()">Refresh</button>
     </form>
-    <ul id="comments-container"></ul>
-    <button type="button" onclick="deleteComments()">Delete comments</button>    
+    <ol id="comments-container"></ol>
+    <form>
+      <select name="like" id="like"></select>
+      <button type="button" onclick="likeComment()">👍</button>
```

Comment:
How about grouping each comment with a thumbs up button? In the HTML right now you can you have a comments-container(plural). Inside this you can have comment-container(singular) (this could be a div element I guess) for each individual comment. Inside the comment-container you can have whatever you want that represents a comment (profile pic, name, comment, thumbs up, report, etc). You can dynamically create these in your JS like you do with appending \<li\> elements. The comment-container would also have an attribute with the entity id from the back-end as discussed in the other comment. There should be a way to get the attribute of the nearest comment-container ancestor when giving a thumbs up so you can get the id so you can send to the back-end.

In the end, this would make your UI comment creator and structure cleaner.

Reply:
That indeed would make the interface nicer and no one would be confused by duplicated comments. Thanks for the suggestion! Will implement it.

Reply:
Done.